readthedocs/projects/migrations/0077_remote_repository_data_migration.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.24 on 2021-06-17 12:14
+
+from django.db import migrations
+
+
+def migrate_data(apps, schema_editor):
+    RemoteRepository = apps.get_model('oauth', 'RemoteRepository')
+    queryset = RemoteRepository.objects.filter(project__isnull=False).select_related('project')
```

Comment:
```suggestion
    queryset = RemoteRepository.objects.filter(project__isnull=False).select_related('project').only('pk', 'project')
```

Maybe we can use `only()` here.